.gitignore

```diff
@@ -7,6 +7,7 @@
 /tmp/*
 !/log/.keep
 !/tmp/.keep
+/out
```

Comment:
When i used Intellij, it automatically generate an out folder. I didn't want git to keep track of the folder, so i added that line. will that break anyboday's code?